fastlane/Constants/Constants.rb

```diff
@@ -0,0 +1,153 @@
+# frozen_string_literal: true
+
+class Constants
+  #################
+  #### PROJECT ####
+  #################
+  
+  # Project path
+  def self.PROJECT_PATH
+    './{PROJECT_NAME}.xcodeproj'
+  end
+
+  def self.BUNDLE_ID_STAGING
+    'com.nimblehq.{PROJECT_NAME}'
```

Comment:
```suggestion
    'co.nimblehq.{PROJECT_NAME}'
```
Not sure if we need a different bundle ID for staging like `co.nimblehq.{PROJECT_NAME}.staging`?

Reply:
I think we can use something like this:
```rb
def self.BUNDLE_ID_STAGING
    '{BUNDLE_ID_STAGING}'
end
```
What do you think?